airflow/operator/rbac/auth_proxy_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
For ClusterRoleBinding, this should go in the "cluster" directory rather than the operator directory.